dxr/plugins/xpidl/mappings.py

```diff
@@ -0,0 +1,29 @@
+from dxr.filters import LINE
+from dxr.indexers import STRING_PROPERTY
+
+# Like qualified line needle, but has no qualname
+LINE_NEEDLE = {
+    'type': 'object',
+    'properties': {
+        'name': STRING_PROPERTY,
+        'start': {
+            'type': 'integer',
+            'index': 'no'  # just for highlighting
+        },
+        'end': {
+            'type': 'integer',
+            'index': 'no'
+        }
+    }
+}
+
+mappings = {
+    LINE: {
+        'properties': {
+            'xpidl_var_decl': LINE_NEEDLE,
```

Comment:
Our convention until now has been to use file extensions as the language prefix. Is "idl" perhaps shared with some of the other IDL variants?